security/cert_manager_operator/cert-manager-operator-proxy.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-operator-proxy"]
+= Configuring the egress proxy for the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-proxy
+
+toc::[]
+
+If a cluster-wide egress proxy is configured in {product-title}, Operator Lifecycle Manager (OLM) automatically configures Operators that it manages with the cluster-wide proxy. OLM automatically updates all of the Operator's deployments with the `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` environment variables.
+
+You can inject any CA certificates that are required for proxying HTTPS connections into the {cert-manager-operator}.
```

Comment:
Looks like this sentence is also in the procedure module (modules/cert-manager-proxy-support.adoc) directly below this sentence. Id recommend removing one of them.